encorelab/SCORE-Client
# chore(Code): Merge WISE into SCORE on 2023-10-27

## Changes

Merged WISE into SCORE which includes new WISE code from [2023-10-03 to 2023-10-27](https://github.com/WISE-Community/WISE-Client/compare/10f3da5b4972a75bf9ecba473d7e4d09ad5db33c...87367eedbdd92bde30318e0c4f6bc64193d3cb55)

## Test

- Test with https://github.com/encorelab/SCORE-API/pull/19
- Make sure the new code from WISE works
- Make sure the existing SCORE code works
